ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.qrc

```diff
@@ -1,11 +1,13 @@
 <RCC>
-    <qresource prefix="/Samples/Scenes/DisplayScenesInTabletopAR">
-        <file alias="sample.json">DisplayScenesInTabletopAR.json</file>        
+    <qresource prefix="/Samples/AR/DisplayScenesInTabletopAR">
+        <file alias="sample.json">DisplayScenesInTabletopAR.json</file>
         <file>DisplayScenesInTabletopAR.qml</file>
         <file>DisplayScenesInTabletopAR.h</file>
         <file>DisplayScenesInTabletopAR.cpp</file>
         <file>main.qml</file>
         <file>screenshot.png</file>
         <file>README.md</file>
+        <file>PermissionsHelper.cpp</file>
+        <file>PermissionsHelper.h</file>
```

Comment:
Just for my curiosity, why are source files added to the project resources? Is this normal?

Reply:
Just a mistake of mine when I added them to the project. Thanks for catching it.